docs/RoleNORManager.md

```diff
@@ -0,0 +1,43 @@
+## OBO Role: New Ontology Request (NOR) Manager
```

Comment:
Maybe better to directly add some frontmatter:

```suggestion
---
permalink: /role/nor-manager
title: OBO Role: New Ontology Request (NOR) Manager
---

Reply:
Somehow your suggestions does not work anymore. I have never seen this use of frontmatter, I will just have to trust you it works as intended. can you recreate the suggestion?